monero-wallet-gui.pro

```diff
@@ -1,3 +1,8 @@
+# qml components require at least QT 5.7.0
+lessThan (QT_MAJOR_VERSION, 5) | lessThan (QT_MINOR_VERSION, 7) {
```

Comment:
Doesn't this error out with future Qt versions whose minor version number is less than 7, like Qt 6.3?